src/libraries/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -769,7 +769,25 @@ public void TestPeakWorkingSet64()
         {
             CreateDefaultProcess();
 
-            AssertNonZeroAllZeroDarwin(_process.PeakWorkingSet64);
+            if (OperatingSystem.IsMacOS())
+            {
+                Assert.Equal(0, _process.PeakWorkingSet64);
+                return;
+            }
+
+            // On recent Linux kernels (6.2+) working set can be zero just after the process started.
+            RetryHelper.Execute(() =>
+            {
+                try
+                {
+                    Assert.NotEqual(0, _process.PeakWorkingSet64);
+                }
+                catch
```

Comment:
we should catch only specific exception, so in case a bug is introduced to `PeakWorkingSet64` and it throws different exception it's detected